### DISCUSSION TITLE
[API] List source/destination parameters

### DISCUSSION THREAD

**Discussion by w14** [airbytehq/airbyte#32514] [Connector Questions]
Is there a programmatic way to get the list of possible source types and what parameters you need to pass to them?

**Comment by marcosmarxm** [comment]
Yes, use `source_definitions/list_latest` ([doc](https://airbyte-public-api-docs.s3.us-east-2.amazonaws.com/rapidoc-api-docs.html#post-/v1/source_definitions/list_latest)) but I didn't find a specific endpoint to return the connector configuration. You can double check the API documentation here: https://docs.airbyte.com/api-documentation

[a comment by an automated account is not shown]
